charts/simple-app/values.yaml

```diff
@@ -482,3 +482,69 @@ runbookUrl:
 istio-alerts:
   # -- (`bool`) Whether or not to enable the istio-alerts chart.
   enabled: true
+
+# Container Alerting Rules
+#
+# These rules are designed to provide very basic but critical monitoring for
+# the health of your containers and pods. More specific alerts can be created
+# by you - but these apply to the pods and resources managed by this chart.
+prometheusRules:
+  # -- (`bool`) Whether or not to enable the container rules template
+  enabled: true
+
+  # -- (`map`) Additional custom labels attached to every PrometheusRule
+  additionalRuleLabels: {}
+
+  # -- Monitors Pods for Containers that are terminated either for unexpected
+  # reasons like ContainerCannotRun. If that number breaches the $threshold (1)
+  # for $for (1m), then it will alert.
+  PodContainerTerminated:
```

Comment:
these should probably include an `enabled` flag to allow for granularly turning them off